tools_iuc.yaml

```diff
@@ -1475,6 +1475,10 @@ tools:
 
 # MACHINE LEARNING
 
+  - name: deepmicro
+    owner: iuc
+    tool_panel_section_label: Machine Learning
```

Comment:
It it really Machine Learning? The fact that it uses a certain method is a detail that is not interesting for the user, isn't it?

Reply:
despite its name it really is just an autoencoder technique, it could therefore be used for any kind of features - not limited  to microbiome data at all ... ? therefore, I thought ML would match best. I am thinking about a bachelor/master project where we can make it a bit more generic, i.e. allowing the export of the model and transform on new data, then I would probably rename it just to `autoencoder` ... but I thought it would be fair to name it deepmicro atm to give credit to the original publication ...